shapeshift/web
# fix: hanging native wallet label

## Description

This resets the wallet label on `walletInfo` change, fixing a bug where the previously connected wallet label would still be displayed.

## Notice




## Pull Request Type


## Issue (if applicable)

Closes https://github.com/shapeshift/web/issues/2494

## Risk

None

## Testing

- Connect a KeepKey / native wallet
- Switch to Keplr
- The wallet label should be showing the default "Keplr" label

## Screenshots (if applicable)

Develop

<img width="211" alt="image" src="https://user-images.githubusercontent.com/17035424/185682153-3434babc-758d-42b5-bbfd-fda23969f345.png">

This diff

<img width="154" alt="image" src="https://user-images.githubusercontent.com/17035424/185682179-d9a44010-3c62-4a46-bda4-5ba7e20dad19.png">
